packs/gatsby-build/config.yml

```diff
@@ -0,0 +1,29 @@
+
+# Name of the pack
+name: Gatsby Build
+
+# Description of the pack
+description: |+
+  The template pack allows you to get visibilility into the build time of your Gatsby Site,
```

Comment:
Suggestion + typo fix: `The Gatsby observability pack allows you to get visibility into the build time of your Gatsby Site,`

Reply:
though visibilility is a nice sounding word 😂